db/migrate/20140716131107_create_category_translation_tables.rb

```diff
@@ -112,16 +112,16 @@ def down
     new_categories = []
     PublicBodyCategory.all.each do |category|
       category.locale = category.translation.locale.to_s
-      FastGettext.default_available_locales.each do |locale|
-        if locale.to_s != category.locale
+      AlaveteliLocalization.available_locales.each do |locale|
+        if locale != category.locale
```

Comment:
need to take a closer look at that after rebasing onto `develop`, at present it's denying that either categories or headings have `.locale`